invisible_cities/database/load_db_test.py

```diff
@@ -178,3 +179,47 @@ def test_pmt_noise_frequencies(db):
     assert frequencies.shape[0] == db.nfreqs
     assert frequencies.shape[1] == db.feboxes + 1
     assert np.all(frequencies[:, 0] == freq_expected)
+
+
+def test_RadioactivityData_max_version():
+    '''Check that RadioactivityData returns by default the data with maximum Version number'''
+
+    db_file = "next100"
+
+    # test unique (G4Volume, Isotope)
+    activity, efficiency = DB.RadioactivityData(db_file)
+    assert np.all(activity  .groupby(["G4Volume", "Isotope"]).count() == 1)
+    assert np.all(efficiency.groupby(["G4Volume", "Isotope"]).count() == 1)
+
+    conn = sqlite3.connect(DB.get_db(db_file))
+
+    query = "SELECT * FROM {0} WHERE G4Volume = '{1}' AND Isotope = '{2}'"
+
+    for table, table_name, column in zip((      activity ,    efficiency ),
+                                         (     "Activity",   "Efficiency"),
+                                         ("TotalActivity", "MCEfficiency")):
+        for (g4volume, isotope), req in table.groupby(["G4Volume", "Isotope"]):
+            all_versions = pd.read_sql_query(query.format(table_name, g4volume, isotope), conn)
```

Comment:
I think defining the query string within the loops will be more readable since you can use the f-string formatting.